tests/test.py

```diff
@@ -19,7 +19,8 @@
 
 SUITE = unittest.TestSuite()
 TESTS = [
-    'tests.i18n.test_locale'
+#    'tests.i18n.test_locale',
```

Comment:
Good catch! No there isn't, I just commented them out because I only wanted to run mine. Will fix.